velox/type/Timestamp.h

```diff
@@ -32,13 +32,23 @@ namespace facebook::velox {
 
 struct TimestampToStringOptions {
   enum class Precision : int8_t {
-    kMilliseconds = 3,
-    kNanoseconds = 9,
+    kMilliseconds = 3, // 10^3 milliseconds are equal to one second.
+    kMicroseconds = 6, // 10^6 microseconds are equal to one second.
+    kNanoseconds = 9, // 10^9 nanoseconds are equal to one second.
   };
 
   Precision precision = Precision::kNanoseconds;
 
+  // Whether to add a leading '+' when year is greater than 9999.
+  bool leadingPositiveSign = false;
+
+  // Whether to skip trailing zeros of fractional part.
```

Comment:
nit: might be helpful to add an example; otherwise it is a bit hard to understand this and zeroPaddingYear options.

Reply:
Opened https://github.com/facebookincubator/velox/pull/8250 to address this comment, thanks.